### DISCUSSION TITLE
Order forks lists by date, recent contribution

### DISCUSSION THREAD

**Discussion by myfirstnameispaul** [community/community#14395] [Other Feature Feedback, Questions, & Ideas]
I would like to sort the forks list of a repo by date forked or most recent contribution.

This is would be very useful for abandoned projects that have a huge number of forks to go and find someone who is actively working to maintain the fork.

**Comment by hayguen** [comment]
fully agree to @myfirstnameispaul . would be great to see the message like

> This branch is 65 commits behind steve-m/librtlsdr:master

for every fork .. and probably for ever branch in that fork.

**Reply by Mp4official** [reply]
every fork .. and probably for ever branch in that fork.

**Comment by jumpjack** [comment]
Found this:

https://github.com/useful-forks/useful-forks.github.io

**Comment by Dubfresh** [comment]
Bug
